packages/mdc-fab/_mixins.scss

```diff
@@ -58,7 +58,7 @@ $mdc-fab-icon-enter-duration_: 180ms;
   @include mdc-ripple-radius-bounded;
   @include mdc-elevation(6);
 
-  display: inline-flex;
+  display: inline-block;
```

Comment:
This seems like a heavy-handed change just to fix IE 11. Does Button not have a similar issue? That uses inline-flex as well.

If we change this, `justify-content` no longer has any meaning and I suspect we should add `text-align: center`, but this further confuses me as to how this is fixing something without breaking anything else.